examples/fiverthirtyeight-example/pages/datasets/[datasetName].tsx

```diff
@@ -0,0 +1,130 @@
+import { NextSeo } from 'next-seo';
+import { promises as fs } from 'fs';
+import path from 'path';
+import getConfig from 'next/config';
+import { getProjectReadme, GithubProject } from '@/lib/octokit';
+import remarkGfm from 'remark-gfm';
+import extract from 'remark-extract-frontmatter';
+import { Dataset } from '..';
+import { GetStaticProps } from 'next';
+import { Table } from '@portaljs/components';
+import Breadcrumbs from '@/components/Breadcrumbs';
+import { ReactMarkdown } from 'react-markdown/lib/react-markdown';
+import remarkFrontmatter from 'remark-frontmatter';
+
+export default function DatasetPage({
+  dataset,
+}: {
+  dataset: Dataset & {
+    readme: string | null;
+  };
+}) {
+  return (
+    <>
+      <NextSeo title={`${dataset.name} page`} />
+      <main className="max-w-5xl px-2 prose mx-auto my-8 prose-thead:border-b-4 prose-table:max-w-5xl prose-table:overflow-scroll prose-thead:overflow-scroll prose-tbody:overflow-scroll prose-thead:pb-2 prose-thead:border-zinc-900 prose-th:uppercase prose-th:text-left prose-th:font-light prose-th:text-xs">
+        <Breadcrumbs links={[{ title: dataset.name, href: '' }]} />
+        <h1 className="uppercase mb-0 mt-16">{dataset.name}</h1>
+        <p className="mb-8">
+          <span className="font-semibold">Repository:</span>{' '}
+          <a target="_blank" href={dataset.url}>
+            {dataset.url}
+          </a>
+        </p>
+
+        <h2 className="mb-0 mt-10">FILES</h2>
+        <div className="inline-block min-w-full py-2 align-middle">
+          <table className="min-w-full divide-y divide-gray-300">
+            <thead className="border-b-4 pb-2 border-zinc-900">
+              <tr>
+                <th
+                  className="uppercase text-left font-light text-xs pb-3"
+                  scope="col"
+                >
+                  Name
+                </th>
+              </tr>
+            </thead>
+            <tbody className="divide-y divide-gray-200">
+              {dataset.files?.map((file) => (
+                <tr key={file}>
+                  <td className="whitespace-nowrap text-left py-4 text-sm text-gray-500">
+                    <a href={file}>{file.split('/').slice(-1)}</a>
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        </div>
+        {dataset.files && dataset.files.length > 0 && (
+          <>
+            <h2 className="mb-0 mt-10">PREVIEWS</h2>
+            {dataset.files?.map((file) => (
+              <div key={file} className="preview-table my-8">
```

Comment:
You have to add a title to each table. It can be the name of the file that is being previewed. This context is missing.